src/omero/plugins/render.py

```diff
@@ -353,50 +352,50 @@ def info(self, args):
                     first = False
             finally:
                 ro.close()
-        gateway._assert_unregistered("info")
+        # gateway._assert_unregistered("info")
 
     def copy(self, args):
         client = self.ctx.conn(args)
         gateway = BlitzGateway(client_obj=client)
         self._copy(gateway, args.object, args.target, args.skipthumbs)
-        gateway._assert_unregistered("copy")
+        # gateway._assert_unregistered("copy")
 
     def _copy(self, gateway, obj, target, skipthumbs, close=True):
         """
             close - whether or not to close the source image
         """
         for src_img in self.render_images(gateway, obj, batch=1):
-            try:
-                self._copy_single(gateway, src_img, target, skipthumbs)
-            finally:
-                if close:
-                    src_img._closeRE()
+            # try:
+            self._copy_single(gateway, src_img, target, skipthumbs)
+            # finally:
+            # if close:
+            # src_img._closeRE()
```

Comment:
Can remove commented-out lines and the "close" argument from ```def _copy()```

Reply:
The argument is not used but as soon as the method is available in omero cf https://github.com/openmicroscopy/openmicroscopy/pull/5545
it will.

It makes more sense to keep it even if it is not used for now